core/src/operations/mod.rs

```diff
@@ -442,7 +455,7 @@ impl Operation {
 
             Self::RpPerm    => 0b0101_0000,
             Self::MpVerify  => 0b0101_0010,
-            // <empty>      => 0b0101_0100
+            Self::Expacc    => 0b0101_0100,
```

Comment:
One thing I wanted to ask: is it possible to move this under a different category? For example, under `0b0010_1000`? I remember the constraints for this had to be degree 3 (but don't remember why), and I'm wondering if we can use temp registers to reduce the degree somehow.

Reply:
When we are including the `exp` value at the latest bit to the accumulated value, we are doing the following:

$$
acc = acc * ( (exp - 1) * bit + 1)
$$

We can make it a degree 2 operation if we introduce a register for the calculation of value which needs to be included to the accumulator in the current cycle.